validators/conversation.go

```diff
@@ -0,0 +1,43 @@
+package validators
+
+import (
+	conversationsv1 "accounts-service/protorepo/noted/accounts/v1"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+)
+
+func ValidateCreateConversationRequest(in *conversationsv1.CreateConversationRequest) error {
+	return validation.ValidateStruct(in,
+		validation.Field(&in.GroupId, validation.Required, is.UUID),
+		validation.Field(&in.Title, validation.Required, validation.Length(1, 20)),
```

Comment:
That's a very nice touch that you added length requirements to some fields. It's a great idea.
Two things off the top of my head:
1. We should find a way to advertise this to the rest of the group. For now no one knows that it's limited to 20 characters.
2. 20 characters is a little short? I don't know. We can leave it like that for now.

Reply:
1. You right 20 is small, but easy to change. I will put 30 for the moment.

2. Maybe in ```CreateConversationRequest``` in the protobuf put a field TitleLength which control the length of the title ?

Reply:
I don't think it would make sense to add a field. What would the following request mean?
```json
{
    "title": "General",
    "title_length": 20
}
```
Does this mean the user chooses how long he wants the field to be? Or would the value be constant at 20? Does this mean the user has to put "20" every time he makes a request? I'm not sure I understand :(

What we could do is add an annotation such as those provided by [this protobuf extension](https://github.com/bufbuild/protoc-gen-validate) (check the README.md). Let me know what you think.

Reply:
<img width="590" alt="Screenshot 2022-12-21 at 09 40 29" src="https://user-images.githubusercontent.com/63368264/208800385-0d85a65a-1eb0-464e-b5e2-31050a6d51e0.png">